docs/user/dql/functions.rst

```diff
@@ -2769,6 +2769,46 @@ Another example to show how to set custom values for the optional parameters::
     +----------------------+--------------------------+
 
 
+MATCHPHRASEQUERY
```

Comment:
if it is alias of matchquery_phrase, could you simply the doc and merge them. E.g. https://dev.mysql.com/doc/refman/8.0/en/date-and-time-functions.html#function_current-date

Reply:
Addressed in 6b23d10ff335fc8d4df6d2e5060abad2d9ffbe21